protocol/x/clob/keeper/process_operations.go

```diff
@@ -713,6 +687,21 @@ func (k Keeper) PersistMatchDeleveragingToState(
 				err,
 			)
 		}
+
+		if quoteQuantums, err := k.perpetualsKeeper.GetNetNotional(
+			ctx,
+			perpetualId,
+			new(big.Int).SetUint64(fill.FillAmount),
```

Comment:
Suggestion: use [`ToBigInt`](https://github.com/dydxprotocol/v4-chain/blob/d9fc60643fabfe028afd21e26a0820b3b042d070/protocol/x/subaccounts/types/subaccount.go#L20-L23).

```suggestion
			fill.FillAmount.ToBigInt(),
```

Reply:
This field is a uint64, not serializable int so doesn't have a ToBigInt function 😭